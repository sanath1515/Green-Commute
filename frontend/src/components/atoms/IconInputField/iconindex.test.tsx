/**
 * @jest-environment jsdom
 */

import React from "react";
import Iconindex from "./iconindex";
import { render } from "@testing-library/react";

describe("component renders correctly", () => {
  test("Matches Snapshot", () => {
    const { asFragment } = render(
      <Iconindex size="small" variant="outlined" color="primary" />
    );
    expect(asFragment.name).toMatchSnapshot();
  });
});
