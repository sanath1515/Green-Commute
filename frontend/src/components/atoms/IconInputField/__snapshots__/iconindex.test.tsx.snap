// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component renders correctly Matches Snapshot 1`] = `"asFragment"`;
